cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,48 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+.. highlight:: xml
+
+YARN containers launched by a CDAP application use the default container logback, ``logback-container.xml``,
+packaged with CDAP. This ``logback`` does log rotation once every day at midnight and
+deletes logs older than 14 days. Depending on the use case, the default configuration may be sufficient.
+However, it is also possible to specify a custom ``logback.xml`` for a CDAP application by packaging
+it with the application in the application's ``src/main/resources`` directory.
+The packaged ``logback.xml`` is then used for each container launched by the application.
+
+Writing and packaging a logback.xml
+===================================
+To specify a ``logback.xml`` for use by an application, all that's required is to package the XML file as a resource
+in the application jar file. The ``pom.xml`` of the application, with the application's ``resource`` directory
+containing the ``logback.xml``, might look like this::
+
+  <?xml version="1.0" encoding="UTF-8"?>
+    <project xmlns="http://maven.apache.org/POM/4.0.0"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    ...
+
+      <build>
+        <resources>
+          <resource>
+            <directory>resources</directory>
+          </resource>
+        </resources>
+      </build>
+
+    ...
+
+  </project>
+
+To write a custom logback, you may refer to `LOGBack <http://logback.qos.ch/>`__.
```

Comment:
Even though the logo might say LOGBack, the text says Logback. Since we are using the text here, let's use Logback here.